package.json

```diff
@@ -374,6 +374,17 @@
                     "when": "view == rokuAutomationView && brightscript.rokuAutomationView.autorunOnDeploy",
                     "group": "navigation@2"
                 },
+                {
+                    "command": "extension.brightscript.rokuAutomationView.exportAutomation",
+                    "when": "view == rokuAutomationView",
+                    "group": "navigation@2"
+                },
+                {
+                    "command": "extension.brightscript.rokuAutomationView.loadAutomation",
```

Comment:
should this command name be `import` instead of "load" to follow the typical "import/export" language?